local-dev-env/serviceConfig.json

```diff
@@ -17,6 +17,9 @@
   "terminologyService": {
     "baseUrl": "http://localhost:8081/api"
   },
+  "excelExportService": {
+    "baseUrl": "http://localhost:3000"
```

Comment:
I think there is /excel at the end?

Reply:
Yes, I added that in code in case we have different endpoints for that service.